ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -125,6 +125,10 @@ message Commands {
   // Additional contracts used to resolve contract & contract key lookups.
   // Optional
   repeated DisclosedContract disclosed_contracts = 17;
+
+  // Must be a valid domain ID
+  // Optional
```

Comment:
same question here, I thought the plan was to make it mandatory.

Reply:
same: it would be mandatory from your perspective

Reply:
I've added a comment clarifying that this is mandatory in case multi domain api. Also added a comment that this will be mandatory for 3.0